wallet-web-app/src/stylesheets/scss/components/sk-dropdown-menu.scss

```diff
@@ -0,0 +1,60 @@
+.sk-dropdown-menu {
+  cursor: pointer;
+  color: #93b0c1;
+  > * {
+     font-family: ProximaNova;
```

Comment:
@tmchedlishvili  change this to ProximaNovaRegular - check variables.scss file for font names..